src/main/java/org/reaktivity/nukleus/http_cache/internal/HttpCacheCounters.java

```diff
@@ -44,6 +45,7 @@ public HttpCacheCounters(
 
         this.requests = supplyCounter.apply("http-cache.requests");
         this.requestsCacheable = supplyCounter.apply("http-cache.requests.cacheable");
+        this.requestsCacheableToOrigin = supplyCounter.apply("http-cache.requests.cacheable.to.origin");
```

Comment:
Rename counter to `"http-cache.group.requests.cacheable"`.
Rename field to `groupRequestsCacheable`.